handler/student.go

```diff
@@ -0,0 +1,39 @@
+package handler
+
+import (
+	"github.com/FrosTiK-SD/auth/constants"
+	"github.com/FrosTiK-SD/auth/controller"
+	"github.com/FrosTiK-SD/auth/model"
+	studentModel "github.com/FrosTiK-SD/models/student"
+	"github.com/gin-gonic/gin"
+	"github.com/gin-gonic/gin/binding"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func (h *Handler) HandlerUpdateStudentDetails(ctx *gin.Context) {
+	studentCollection := h.MongikClient.MongoClient.Database(constants.DB).Collection(constants.COLLECTION_STUDENT)
```

Comment:
Why??

Reply:
Right now ignoring this as there is no good way for Mongik to return nil results in FindOne ... rest wherever I could, I have replaced DB queries with Mongik